ircbot/plugins/links.py

```diff
@@ -36,8 +44,7 @@ def link_title_parse_hook(bot, channel, sender, message):
                     return
 
                 soup = BeautifulSoup(r.text, 'html.parser')
-                title = soup.head.title.text.strip()
-                title = title.replace("\n", "")
+                title = truncate_title(soup.head.title.text.strip())
```

Comment:
Can we still leave this as two lines - one doing `soup.head.title.text.strip()` and the other doing `title = truncate_title(title)` - I feel it's a bit more readable?